Sources/NimbleTests/Matchers/PostNotificationTest.swift

```diff
@@ -1,7 +1,20 @@
 import XCTest
 import Nimble
+import Foundation
+
+class PostNotificationTest: XCTestCase, XCTestCaseProvider {
```

Comment:
This test class should also be added to the `XCTMain` invocation in `Sources/NimbleTests/main.swift`